GitUI/CommandsDialogs/BrowseDialog/FormOpenDirectory.cs

```diff
@@ -95,16 +95,13 @@ public static GitModule OpenModule(IWin32Window owner, [CanBeNull] GitModule cur
         private void LoadClick(object sender, EventArgs e)
         {
             _NO_TRANSLATE_Directory.Text = _NO_TRANSLATE_Directory.Text.Trim();
-            if (Directory.Exists(_NO_TRANSLATE_Directory.Text))
+            if ((_chosenModule = OpenGitRepository(_NO_TRANSLATE_Directory.Text, RepositoryHistoryManager.Locals)) != null)
```

Comment:
Just my 2 cents: I don't like assingments in `if` conditions.

Reply:
Nw, easy fix.